pkg/authenticator/authenticator.go

```diff
@@ -119,12 +112,23 @@ func (auth *Authenticator) Login() error {
 
 	log.InfoLogger.Printf(log.CAKC007I, auth.Config.Username)
 
-	csrRawBytes, err := auth.GenerateCSR()
+	// We want to let hosts authenticate with Conjur from anywhere in the policy
```

Comment:
Seems this comment should be in the ticket (which number should be part of this PR name)

Reply:
This comment may be also in the ticket but we also have it in the code for the next reader who will never look for a ticket.

Regarding the PR name - we do not have the ticket in it but we do have it in the description. I'll add it.